app/types/state/shieldsPannelState.ts

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BlockOptions, BlockTypes } from '../other/blockTypes'
+
+// TODO  @bbondy which are required?
```

Comment:
I prefer all unless it's a problem, is it?

Reply:
not a problem